keyboards/j80/info.json

```
@@ -0,0 +1,202 @@
+{
+  "keyboard_name": "J80",
+  "url": "",
+  "maintainer": "JER",
+  "width": 18.25,
+  "height": 6.25,
+  "layouts": {
+    "LAYOUT_tkl_ansi": {
```

Your ANSI and ISO layout macros are actually both configured for split right Shift.

```suggestion
    "LAYOUT_tkl_ansi_split_rshift": {
```